src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.wasm.cs

```diff
@@ -99,7 +99,19 @@ void OnJsonChanged(string updatedJson, string updatedCacheKey)
 				}
 				else
 				{
-					var documentPath = Windows.Storage.Helpers.AssetsPathBuilder.BuildAssetUri(UriSource?.PathAndQuery);
+					Uri _result = null;
+					var documentPath = string.Empty()
+					
+					Uri.TryCreate(UriSource.ToString(), UriKind.Absolute, out _result);
```

Comment:
You can use the inline variable declaration:

```suggestion
					var documentPath = string.Empty()
					
					if(Uri.TryCreate(UriSource.ToString(), UriKind.Absolute, out var result))
```